packages/labs/router/src/routes.ts

```diff
@@ -118,10 +123,12 @@ export class Routes implements ReactiveController {
 
   constructor(
     host: ReactiveControllerHost & HTMLElement,
-    routes: Array<RouteConfig>
+    routes: Array<RouteConfig>,
+    fallbackRoute?: RouteConfig
```

Comment:
Maybe just `fallback` for brevity? Let's ask some of the team.

Reply:
`fallback` seems good.